docs/changelog.md

```diff
@@ -9,6 +9,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/plugins/dependencies/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/.config.yml)
 
+* **service-pocket**
```

Comment:
Fly-by: might be wise to add this to the YAML files for the metrics that are affected by this change so that folks know that, up to version X, this meant something, after version X it means something else. This is necessary since we receive data from both newer and older versions.

Reply:
@Dexterp37 Updated. Can you confirm that the updated documentation looks okay?

Reply:
Thanks! @Mugurell if the build system doesn't complain, this is good to land :)